package.json

```diff
@@ -2,7 +2,7 @@
   "name": "hfx",
   "version": "0.0.1",
   "description": "Enhance your HF experience, a native browser extension.",
-  "main": "src/_core/HFX.js",
+  "main": "src/HFX.js",
```

Comment:
Did I miss something or should this still be in the core?

Reply:
The path for `HFX.js` is `src/HFX.js`. The structure was changed a while ago and this property wasn’t updated. The HFX module bundles all the core modules together.